src/deps/libuv.zig

```diff
@@ -1344,14 +1344,50 @@ pub const struct_uv_process_s = extern struct {
     u: union_unnamed_424,
     endgame_next: [*c]uv_handle_t,
     flags: c_uint,
-    exit_cb: ?*const fn ([*c]struct_uv_process_s, i64, c_int) callconv(.C) void,
+    exit_cb: ?*const fn ([*c]uv_process, i64, c_int) callconv(.C) void,
     pid: c_int,
     exit_req: struct_uv_process_exit_s,
     unused: ?*anyopaque,
     exit_signal: c_int,
     wait_handle: HANDLE,
     process_handle: HANDLE,
     exit_cb_pending: u8,
+
```

Comment:
these functions shouldn't need to alignCast

but i am glad we have more of these wrappers now.
can we not use `HandleMixin` for some of these directly?